### PR TITLE
Enable conversion from PCLPointCloud2 to PointCloud<PointT> for clouds with greater than 2^28 points 

### PR DESCRIPTION
**Pull request by mudskipper75**
Changed std::uint32_t to std::size_t on lines 208 and 211 of conversions.h to enable conversion from PCLPointCloud2 to PointCloud<PointT> for clouds with more than 2^28 points. Related to issue #4326

### CODE REVIEW

File: common/include/pcl/conversions.h [hunk 1]
```diff
@@ -205,10 +205,10 @@ namespace pcl
     else
     {
       // If not, memcpy each group of contiguous fields separately
-      for (std::uint32_t row = 0; row < msg.height; ++row)
+      for (std::size_t row = 0; row < msg.height; ++row)
```
Comment (kunaltyagi) [inline common/include/pcl/conversions.h:208]: @haritha-j Does one of your PR address this change?

Comment (mudskipper75) [reply common/include/pcl/conversions.h:208]: I issued a pull request on Monday related to the loading and saving of files. The one I have put in just now relates to conversion to templated types. I have used the same fork for the pull request though. If this is an issue I can create another fork an put the request in again.

Comment (haritha-j) [reply common/include/pcl/conversions.h:208]: @kunaltyagi in #4199 i changed `height` of `pointCloud2` to `index_t`. I have also replaced some instances of `std::size_t` with `index_t`. I basically updated the instances where `sign-compare` warnings came up. This file doesn't seem to be touched there.

**Comment by mvieth** [comment]
@mudskipper75 It seems like the two commits that were included in your other pull request are in this one, too. You will have to remove them. A rebase with `git rebase -i HEAD~3` and then `drop` in front of the commits you want to remove should do the trick.

**Comment by mudskipper75** [comment]
> @mudskipper75 It seems like the two commits that were included in your other pull request are in this one, too. You will have to remove them. A rebase with `git rebase -i HEAD~3` and then `drop` in front of the commits you want to remove should do the trick. Edit: and then you have to force-push the new commit to your fork, with `git push --force`. Careful, that overwrites the commit history, but in this case, that's what you want. (I'm not sure if this is new to you or not, if you already know this, never mind)

I couldn't get it the forced push work on the system so I have decided to close the pull. I will create another fork and issue a pull request on that one.

**Pull request closed without merging**